src/QMCWaveFunctions/Fermion/DiracDeterminant.cpp

```diff
@@ -44,7 +44,11 @@ void DiracDeterminant<DU_TYPE>::set(int first, int nel, int delay)
 {
   FirstIndex = first;
   ndelay     = delay;
-  resize(nel, nel);
+
+  resize(nel, Phi->OrbitalSetSize);
```

Comment:
I believe you need to do this change to make the code working.
On the other hand, I feel we should not resize all the matrices in DiracDeterminant to Phi->OrbitalSetSize.
Could you describe what you experienced?

Reply:
If you leave the resize arguments to both be 'nel' the problem occurs when the evaluate functions inside LCAOrbitalSet.cpp are called. LCAOrbitalSet member variable OrbitalSetSize will cause lines such as **std::copy_n(temp.data(0), OrbitalSetSize, psi.data());**  to copy too many elements into psi.data(). So either all the matrices in DiracDeterminant need to be resized or the evaluate functions in LCAOrbitalSet need a different way of determining how many elements to copy into psi.data() argument

Reply:
We need to fix the evaluate functions to respect result vector size, namely the number of SPOs needed can be smaller than the number of SPOs SPOSet can provide.